src/main/java/net/sf/jabref/logic/exporter/BibTeXMLExportFormat.java

```diff
@@ -0,0 +1,247 @@
+        Objects.requireNonNull(entries);
+        if (entries.isEmpty()) { // Only export if entries exist
+            return;
+        }
+
+        File file = new File();
+        for (BibEntry bibEntry : entries) {
+            Entry entry = new Entry();
+
+            Optional<String> citeKey = bibEntry.getCiteKeyOptional();
+            if (citeKey.isPresent()) {
+                entry.setId(citeKey.get());
+            }
+
+            String type = bibEntry.getType().toLowerCase(ENGLISH);
+            switch (type) {
+                case "article":
+                    parse(new Article(), bibEntry, entry);
+                    break;
+                case "book":
+                    parse(new Book(), bibEntry, entry);
+                    break;
+                case "booklet":
+                    parse(new Booklet(), bibEntry, entry);
+                    break;
+                case "conference":
+                    parse(new Conference(), bibEntry, entry);
+                    break;
+                case "inbook":
+                    parseInbook(new Inbook(), bibEntry, entry);
+                    break;
+                case "incollection":
+                    parse(new Incollection(), bibEntry, entry);
+                    break;
+                case "inproceedings":
+                    parse(new Inproceedings(), bibEntry, entry);
+                    break;
+                case "mastersthesis":
+                    parse(new Mastersthesis(), bibEntry, entry);
+                    break;
+                case "manual":
+                    parse(new Manual(), bibEntry, entry);
+                    break;
+                case "misc":
+                    parse(new Misc(), bibEntry, entry);
+                    break;
+                case "phdthesis":
+                    parse(new Phdthesis(), bibEntry, entry);
+                    break;
+                case "proceedings":
+                    parse(new Proceedings(), bibEntry, entry);
+                    break;
+                case "techreport":
+                    parse(new Techreport(), bibEntry, entry);
+                    break;
+                case "unpublished":
+                    parse(new Unpublished(), bibEntry, entry);
+                    break;
+                default:
+                    LOGGER.warn("unexpected type appeared");
+                    break;
+            }
+            file.getEntry().add(entry);
+        }
+        createMarshallerAndWriteToFile(file, resultFile);
+    }
+
+    private void createMarshallerAndWriteToFile(File file, String resultFile) throws SaveException {
+        try {
+            if (context == null) {
+                context = JAXBContext.newInstance(File.class);
+            }
+            Marshaller marshaller = context.createMarshaller();
+            marshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, Boolean.TRUE);
+
+            marshaller.marshal(file, new java.io.File(resultFile));
```

Comment:
not wrong, but there is also a class File that is generated by JAXB. Thats is why this is needed.